src/main/java/com/minecolonies/blocks/BlockHutField.java

```diff
@@ -0,0 +1,239 @@
+package com.minecolonies.blocks;
+
+import com.minecolonies.MineColonies;
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.creativetab.ModCreativeTabs;
+import com.minecolonies.inventory.InventoryField;
+import com.minecolonies.lib.Constants;
+import com.minecolonies.tileentities.ScarecrowTileEntity;
+import com.minecolonies.util.LanguageHandler;
+import net.minecraft.block.BlockContainer;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.properties.PropertyDirection;
+import net.minecraft.block.state.BlockState;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.*;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+/**
+ * The class handling the fieldBlocks, placement and activation.
+ */
+public class BlockHutField extends BlockContainer
+{
+    /**
+     * Hardness of the block.
+     */
+    private static final float HARDNESS   = 10F;
+
+    /**
+     * Resistance of the block.
+     */
+    private static final float RESISTANCE = 10F;
+
+    /**
+     * The position it faces.
+     */
+    private static final PropertyDirection FACING = PropertyDirection.create("FACING", EnumFacing.Plane.HORIZONTAL);
+
+    /**
+     * Start of the collision box at y.
+     */
+    private static final double BOTTOM_COLLISION = 0.0;
+
+    /**
+     * Start of the collision box at x and z.
+     */
+    private static final double START_COLLISION = 0.1;
+
+    /**
+     * End of the collision box.
+     */
+    private static final double END_COLLISION = 0.9;
+
+    /**
+     * Height of the collision box.
+     */
+    private static final double HEIGHT_COLLISION = 2.5;
+
+    /**
+     * Constructor called on block placement.
+     */
+    BlockHutField()
+    {
+        super(Material.wood);
+        initBlock();
+    }
+
+    /**
+     * Getter of the description of the blockHut.
+     * @return the name as a String.
+     */
+    public String getName()
+    {
+        return "blockHutField";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS3400)